pkg/controller/clusterinstallation/mattermost.go

```diff
@@ -226,6 +226,11 @@ func (r *ReconcileClusterInstallation) launchUpdateJob(
 		},
 	}
 
+	// We dont need to validate the readiness/liveness for this short live job
+	// if the job fails it will get another later.
+	job.Spec.Template.Spec.Containers[0].LivenessProbe = nil
+	job.Spec.Template.Spec.Containers[0].ReadinessProbe = nil
```

Comment:
This is a little dangerous as we are referencing `Containers[0]` without checking the length of the slice. It should be fine because we should always have at least one container, but maybe it makes sense to do something like this instead?

```
for i := range job.Spec.Template.Spec.Containers {
  	job.Spec.Template.Spec.Containers[i].LivenessProbe = nil
	job.Spec.Template.Spec.Containers[i].ReadinessProbe = nil
}
```

Reply:
makes sense

Reply:
done @gabrieljackson PTAL